src/nodejs_scripts/example_gen.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
```

Comment:
If this is used solely for testing, this should live in `test` directory imo. `src` should only contain what will be used to construct binary.

Reply:
good point, will move